libzerocash/Address.cpp

```diff
@@ -11,24 +11,35 @@
  * @copyright  MIT license (see LICENSE file)
  *****************************************************************************/
 
-#include <cryptopp/osrng.h>
-using CryptoPP::AutoSeededRandomPool;
+#include "Zerocash.h"
+#include "Address.h"
 
-#include <cryptopp/eccrypto.h>
-using CryptoPP::ECP;
-using CryptoPP::ECIES;
+namespace libzerocash {
 
-#include <cryptopp/oids.h>
-namespace ASN1 = CryptoPP::ASN1;
+PrivateAddress::PrivateAddress() {
```

Comment:
(Does not block this PR.) I dislike the pattern of uninitialized construction, then subsequent mutation, so as a followup to this PR I want to nuke this constructor, then fix every call site that uses it.